patchwork/models.py

```diff
@@ -499,6 +502,10 @@ class Patch(SubmissionMixin):
         null=True,
         on_delete=models.CASCADE,
     )
+    planning_to_review = models.ManyToManyField(
+        User, through='PatchReviewIntention', related_name='planning_to_review'
+    )
+    has_planned_review = models.BooleanField(default=False)
```

Comment:
Do we need this? Can't we simply check if the `planning_to_review` set is empty?